subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -617,18 +617,29 @@ uint32_t ull_scan_is_enabled(uint8_t handle)
 {
 	struct ll_scan_set *scan;
 
+	/* NOTE: BIT(0) - passive scanning enabled
+	 *       BIT(1) - active scanning enabled
+	 *       BIT(2) - initiator enabled
+	 *       BIT(3) - periodic sync create active
+	 */
```

Comment:
These values should really be defines or an enum. That would make the check in ull_filter.c a lot easier to understand.

Reply:
will rework on this.

That said, these fixes related to HCI are for newer tests, I wish to review the test cases too, just to make sure if they reflect what core spec state.

Reply:
@wopu-ot please re-review.